whois-endtoend/src/test/groovy/net/ripe/db/whois/spec/update/InetnumStatusChildSpec.groovy

```diff
@@ -4087,4 +4087,100 @@ class InetnumStatusChildSpec extends BaseQueryUpdateSpec {
         ack.warningSuccessMessagesFor("Modify", "[inetnum] 192.168.200.0 - 192.168.200.127") ==
                 ["inetnum parent has incorrect status: ASSIGNED PA"]
     }
+
+    def "modify parent ALLOCATED PA to ALLOCATED-ASSIGNED PA, child status ASSIGNED PA"() {
```

Comment:
can you create next test?
we have ALLOCATED-PA -> ASSIGNED-PA
And the user modify ASSIGNED-PA TO ALLOCATED-ASSIGNED PA -> ERROR because parent can only be ALLOCATED_UNSPECIFIED

And the same when OK? when the parent is ALLOCATED_UNSPECIFIED

And when override?? 

I think that we cover this when creating but not when modifying